compose/cli/main.py

```diff
@@ -238,8 +240,9 @@ class TopLevelCommand(object):
       version            Show the Docker-Compose version information
     """
 
-    def __init__(self, project, options=None):
+    def __init__(self, project, config=None, options=None):
```

Comment:
Please put new arguments at the end as this ensures that people using the arguments positionally aren't broken.